junit-jupiter-params/src/main/java/org/junit/jupiter/params/provider/CsvFileSource.java

```diff
@@ -69,4 +69,10 @@
 	 */
 	char delimiter() default ',';
 
+	/**
+	 * The number of lines to skip parsing. Could be used for skipping header lines.
+	 *
+	 * <p>Defaults to {@code 0}</p>
+	 */
+	int skipLines() default 0;
```

Comment:
I'm tempted to say...

_Please rename to `numLinesToSkip`._

... but I'll have to ponder it a bit more once I see it in action. 😉